src/context/products-context.js

```diff
@@ -2,18 +2,21 @@
 import { createContext, useContext, useReducer } from "react";
 
 // All local imports
-import { productsReducerFn } from "../reducer/productsReducerFn.jsx";
+import {
+    productsInitialState,
```

Comment:
put the initial state in the context only, and then use the reducer to mutate it

Reply:
Done